spec/models/post_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
```

Comment:
- [ ] Kindy check your post model, I encountered this error when running the tests:
![image](https://user-images.githubusercontent.com/61408860/190589255-cd6d07ad-ac52-40bc-bfc0-4753312993d9.png)